public/js/editor.js

```diff
@@ -1,9 +1,23 @@
 (function() {
     function editorMain() {
```

Comment:
What is stopping this function from being called multiple times?

Reply:
loading the system.js file in a SPA website